generated/nidaqmx/_install_daqmx.py

```diff
@@ -80,8 +90,34 @@ def _get_daqmx_installed_version() -> Optional[str]:
             raise click.ClickException(
                 f"An OS error occurred while getting the installed NI-DAQmx version.\nDetails: {e}"
             ) from e
+    elif sys.platform.startswith("linux"):
+        try:
+            distribution = distro.id()
+            _logger.debug("Checking for installed NI-DAQmx version")
+            commands_info = LINUX_COMMANDS[distribution]
+            query_command = commands_info.get_daqmx_version
+            query_output = subprocess.run(query_command, stdout=subprocess.PIPE, text=True).stdout
+
+            if distribution == "ubuntu":
+                version_match = re.search(r"ii\s+ni-daqmx\s+(\d+\.\d+\.\d+)", query_output)
+            elif distribution == "opensuse" or distribution == "rhel":
+                version_match = re.search(r"ni-daqmx-(\d+\.\d+\.\d+)", query_output)
+            else:
+                raise click.ClickException(f"Unsupported distribution '{distribution}'")
+            if version_match is None:
+                return None
+            else:
+                installed_version = version_match.group(1)
+                _logger.info("Found installed NI-DAQmx version: %s", installed_version)
+                return installed_version
```

Comment:
The none case is already handled

```suggestion
            installed_version = version_match.group(1)
            _logger.info("Found installed NI-DAQmx version: %s", installed_version)
            return installed_version
```

Reply:
How is the none case already handled?

Reply:
oh, I was thinking of the else case raising, but I guess the re.match could return None. dropping.

Reply:
@dharaniprakashkm Please revert this change.